pom.xml

```diff
@@ -98,6 +98,11 @@
       <artifactId>support-log-formatter</artifactId>
       <version>1.3</version>
     </dependency>
+    <!-- plugin dependencies -->
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>commons-lang3-api</artifactId>
```

Comment:
Would prefer to drop this dep and just replace usages of `StringUtils` with simple equivalents from the Java Platform API.

Reply:
There is one place that uses a complex function: `StringUtils.splitByCharacterTypeCamelCase(camelCase)`

Reply:
https://github.com/apache/commons-lang/blob/7302e9a308f0a5d59f17f72b6f5326e76cd71e3a/src/main/java/org/apache/commons/lang3/StringUtils.java#L7196-L7226 FTR. I think https://github.com/jenkinsci/support-core-plugin/blob/a2a9f06eed6188fb78f636ad9b2e6ba5c048ea88/src/main/java/com/cloudbees/jenkins/support/impl/LoadStats.java#L318-L328 could just be deleted and the code name used directly.